.rubocop.yml

```diff
@@ -139,7 +139,7 @@ RSpec/FilePath:
 
 RSpec/DescribeClass:
   Exclude:
-    - "spec/config/initializers/csv_types_spec.rb"
+    - "spec/config/initializers/**/*"
```

Comment:
I'd strongly recommend you not use **/* since you _should_ want to know if new files in `initializers` have a rubocop problem. If you want to exclude this new one, `common.rb`, a line specifying that would be preferred to the wildcard.

Reply:
changed